packages/cxl-ui/src/components/cxl-star-rating.js

```diff
@@ -0,0 +1,40 @@
+import '@conversionxl/cxl-lumo-styles';
+import { IronStarRating } from '@cwmr/iron-star-rating';
+
+export class CXLStarRating extends IronStarRating {
+  static get is() {
+    return 'cxl-star-rating';
+  }
+
+  _getUniqueId() {
+    return `cxl-star-rating_${this.id}`;
+  }
+
+  _saveState(value) {
+    if (value !== false && !Number.isNaN(value)) {
+      localStorage.setItem(this._getUniqueId(), value);
```

Comment:
So we have client-side storage, but what's the bigger plan for server-side rating collection system?

Reply:
This should be handled via `value-changed` custom event handler, which should send POST request to the REST endpoint:
https://www.loom.com/share/c70e72c60104425087dfc981c79eb54f
The example provided in the last commit: https://github.com/conversionxl/aybolit/pull/88/commits/378509b3f74bdd84cb56919d89bc6511a56dba58
A custom event is already provided by the original `iron-star-rating` component.